Web/Pages/CCGMember/Compare/Delta.cshtml.cs

```diff
@@ -0,0 +1,308 @@
+        public JobCompetencyRating[] curratings3 { get; set; }
+        public JobCompetencyRating[] objratings3 { get; set; }
+        public JobCompetencyRating[] curratings4 { get; set; }
+        public JobCompetencyRating[] objratings4 { get; set; }
+        public JobCompetencyRating[] allobjratings { get; set; }
+        [BindProperty]
+        public List<JobCompetencyRating> matchingcomp1 { get; set; }
+        [BindProperty]
+        public List<SharedJobCompetencyRating> diffcomp1 { get; set; }
+        [BindProperty]
+        public List<JobCompetencyRating> matchingcomp2 { get; set; }
+        [BindProperty]
+        public List<SharedJobCompetencyRating> diffcomp2 { get; set; }
+        [BindProperty]
+        public List<JobCompetencyRating> matchingcomp3 { get; set; }
+        [BindProperty]
+        public List<SharedJobCompetencyRating> diffcomp3 { get; set; }
+        [BindProperty]
+        public List<JobCompetencyRating> matchingcomp4 { get; set; }
+        [BindProperty]
+        public List<SharedJobCompetencyRating> diffcomp4 { get; set; }
+        [BindProperty]
+        public List<JobCertificate> matchcert { get; set; }
+        [BindProperty]
+        public List<JobCertificate> diffcert { get; set; }
+        public DeltaModel(ILogger<DeltaModel> logger, JobPositionService jobcompetencyService)
+        {
+            //_logger = logger;
+            _jobpositionService = jobcompetencyService;
+        }
+        public async Task OnGetAsync(int positionid, int obj)
+        {
+            curcertificates = await _jobpositionService.GetJobCertificatesById(positionid);
+            objcertificates = await _jobpositionService.GetJobCertificatesById(obj);
+            curposition = await _jobpositionService.GetJobPosition(positionid);
+            objposition = await _jobpositionService.GetJobPosition(obj);
+            curratings1 = await _jobpositionService.GetJobCompetencyRatingsByTypeId(positionid, 1);
+            objratings1 = await _jobpositionService.GetJobCompetencyRatingsByTypeId(obj, 1);
+            curratings2 = await _jobpositionService.GetJobCompetencyRatingsByTypeId(positionid, 2);
+            objratings2 = await _jobpositionService.GetJobCompetencyRatingsByTypeId(obj, 2);
+            curratings3 = await _jobpositionService.GetJobCompetencyRatingsByTypeId(positionid, 3);
+            objratings3 = await _jobpositionService.GetJobCompetencyRatingsByTypeId(obj, 3);
+            allobjratings = await _jobpositionService.GetJobCompetencyRatings(obj);
+            matchingcomp1 = new List<JobCompetencyRating>();
+            diffcomp1 = new List<SharedJobCompetencyRating>();
+            matchingcomp2 = new List<JobCompetencyRating>();
+            diffcomp2 = new List<SharedJobCompetencyRating>();
+            matchingcomp3 = new List<JobCompetencyRating>();
+            diffcomp3 = new List<SharedJobCompetencyRating>();
+            
+            foreach (var ob in allobjratings)
```

Comment:
This looks like a query. Should it be moved to the query project and given an API endpoint?